src/renderer/component/expandable/view.jsx

```diff
@@ -0,0 +1,57 @@
+// @flow
+import React, { PureComponent, Node } from 'react';
+import classnames from 'classnames';
+import Button from 'component/button';
+
+// Note:
+// When we use this in other parts of the app, we will probably need to
+// add props for collapsed height
+
+type Props = {
+  children: Node | Array<Node>,
+};
+
+type State = {
+  expanded: boolean,
+};
+
+export default class Expandable extends PureComponent<Props, State> {
+  constructor() {
+    super();
+
+    this.state = {
```

Comment:
Can also do:
```
class MyComponent extends PureComponent {
  state = {};
}
```

and use arrow fn's or bind syntax https://babeljs.io/blog/2015/05/14/function-bind